projects/aca-settings/tsconfig.lib.prod.json

```diff
@@ -2,7 +2,12 @@
 {
   "extends": "./tsconfig.lib.json",
   "compilerOptions": {
-    "declarationMap": false
+    "declarationMap": false,
+    "paths": {
```

Comment:
why this paths here? I think it was not using that before

Reply:
Because we were not building these libraries before, if we would like to build `aca-settings` with dependancies, these dependancies need to be available/built for this library.

e.g. if we would left `@alfresco/aca-shared` as `projects/aca-shared/...`, then we would get an error, that angular cannot build `aca-settings` with files located outside of the library root
(if this would be possible, then it would bundle the `aca-shared` within the library, which is also not what we need)